src/features/Home/components/ModelsByStatusGroup/index.tsx

```diff
@@ -0,0 +1,150 @@
+
+  const [currentStatus, setCurrentStatus] = useState(
+    ALL_COLUMNS_BY_STATUS_GROUP[0]
+  );
+
+  const {
+    data: modelPlansByStatusGroup,
+    loading,
+    error
+  } = useGetModelPlansByStatusGroupQuery({
+    variables: {
+      statusGroup: currentStatus
+    }
+  });
+
+  const data = useMemo(() => {
+    return (modelPlansByStatusGroup?.modelPlansByStatusGroup ??
+      []) as ModelPlansType[];
+  }, [modelPlansByStatusGroup?.modelPlansByStatusGroup]);
+
+  const statusNavs = ALL_COLUMNS_BY_STATUS_GROUP.map(statusGroup => (
+    <Button
+      type="button"
+      key={statusGroup}
+      onClick={() => setCurrentStatus(statusGroup)}
+      className={classNames('usa-nav__link margin-left-neg-2 margin-right-2', {
+        'usa-current': currentStatus === statusGroup
+      })}
+      unstyled
+    >
+      <span>
+        {customHomeT(
+          `settings.${ViewCustomizationType.MODELS_BY_STATUS_GROUP}.status.${statusGroup}.label`
+        )}
+      </span>
+    </Button>
+  ));
+
+  if (error) {
+    return <Alert type="error">{customHomeT('fetchError')}</Alert>;
+  }
+
+  return (
+    <div className="models-by-solutions">
+      {!isTablet && (
+        <Header
+          basic
+          extended={false}
+          className="margin-bottom-4 models-by-solutions__nav-container"
+        >
+          <div className="usa-nav-container padding-0">
+            <PrimaryNav
+              items={statusNavs}
+              mobileExpanded={false}
+              className="flex-justify-start margin-0 padding-0"
+            />
+          </div>
+        </Header>
+      )}
+
+      {isTablet && (
+        <div className="maxw-mobile-lg">
+          <Select
+            id="solutionKey"
+            name="solutionKey"
+            value={currentStatus}
+            onChange={(e: React.ChangeEvent<HTMLSelectElement>) =>
+              setCurrentStatus(e.currentTarget.value as ModelPlanStatusGroup)
+            }
+            className="margin-bottom-4 text-primary text-bold"
+          >
+            {ALL_COLUMNS_BY_STATUS_GROUP.map(statusGroup => {
+              return (
+                <option key={statusGroup} value={statusGroup}>
+                  {customHomeT(
+                    `settings.${ViewCustomizationType.MODELS_BY_STATUS_GROUP}.status.${statusGroup}.label`
+                  )}
+                </option>
+              );
+            })}
+          </Select>
+        </div>
+      )}
+      {loading && <PageLoading testId="models-by-status" />}
+
+      {!loading && data.length > 0 ? (
+        <ModelDetailsTable
+          models={data}
+          hiddenColumns={HIDDEN_COLUMNS_BY_STATUS_GROUP[currentStatus]}
+          canSearch={data.length > 4}
+        />
+      ) : (
+        <Alert
+          type="info"
+          heading={customHomeT(
+            `settings.${ViewCustomizationType.MODELS_BY_STATUS_GROUP}.status.${currentStatus}.noResultsHeading`
+          )}
+        />
+      )}
+    </div>
```

Comment:
Sorry I missed this but there should be no heading/bold.  This need to be the child of the alert to match figma

<img width="866" height="191" alt="Screenshot 2025-11-07 at 1 08 11 PM" src="https://github.com/user-attachments/assets/2afff7d4-9d2c-4fdf-9452-27f1d3d401dc" />

Reply:
Good catch! TY!